backend/lcfs/tests/internal_comment/test_internal_comment.py

```diff
@@ -0,0 +1,461 @@
+import pytest
```

Comment:
I believe the goal of the ticket is to have one test file to one python file, but it looks like you combined them. Can you split them back up?

Reply:
I had the same doubt. After discussing with Alex before starting this ticket, we agreed that since these endpoints are simple CRUD, we don’t need to write tests everything (like repos, services, and views).